### ISSUE TITLE
Formalize the Place Type classification vocabulary

### ISSUE THREAD

**Issue by dr-shorthair** [GeoscienceAustralia/Place-Names#22]
Currently visible at https://vocabs.ands.org.au/repository/api/lda/ga/place-type/v1-0/resource?uri=http://pid.geoscience.gov.au/def/voc/ga/PlaceType 

The Groups and Categories would be better implemented as `skos:Collection` rather than broader concepts - they are a way of organizing the classification, they are now classifiers themselves.

**Comment by bellie888** [comment]
That's a question for Shane Crossman. GA and Shane put a lot of work into that vocab, although I know they are looking for your input very much. :)

**Comment by dr-shorthair** [comment]
Looks like Lara Sedgmen was looking after this, but she doesn't appear to be attached to this GitHub repo.

**Comment by bellie888** [comment]
Best to talk to Shane. I do remember one meeting with Lara.

**Comment by dr-shorthair** [comment]
In the current model, the place-type is associated with the `Place` rather than the `PlaceName` but this may not be correct - see https://raw.githack.com/GeoscienceAustralia/Place-Names/master/images/placename-details.png

**Comment by ibastrak** [comment]
Do we need to say it’s hasPlaceNameOrigin?

**Comment by ibastrak** [comment]
Sorry,
hasPlaceNameOrigin or something similar to indicate indigenous connection

**Comment by dr-shorthair** [comment]
Suggest you look at the [ontology documentation](http://linked.data.gov.au/def/placenames) 
and in particular the [summary image](https://raw.githubusercontent.com/GeoscienceAustralia/Place-Names/master/images/placename-details.png) which shows pretty much everything. Then we can re-evaluate the design to verify if it meets all the requirements.

**Comment by bellie888** [comment]
I remember Nick et al talking about this because a Place can have several names, Aboriginal Name, Local Name, Historical name etc, so one placename can have several 'Names'

**Comment by ibastrak** [comment]
Just to clarify:

Lara’s involvement was through GA vocabulary publication process – she is moving them to RVA. I don’t think she would want to be involved in out HitHub repo.

Jenny Long from my section is looking after their context. Not sure if Jenny is part of the group. She is going through a quite quick learning curve with vocabularies.

Any required modifications  for vocabularies would need to go through Joe, Shane and myself – if the GA process is not quite right now, we will need to talk to our geoinformatics section about it.

**Comment by dr-shorthair** [comment]
@bellie888 Indeed. That's why `Place` and `PlaceName` are separated. But that does mean that the various tags must be on the right thing. That's not clear to me that we've got it right yet.

**Comment by dr-shorthair** [comment]
The use of `skos:narrower` to create the taxonomy is incorrect, at least in parts. 
I pulled down the Place-type source from https://vocabs.ands.org.au/viewById/260 
It could be remediated using the following SPARQL process:

1. Find the leaf nodes and give them type `pt:Type` 
```
INSERT { ?s a pt:Type }
WHERE {
	?s a skos:Concept . 
	NOT EXISTS { ?s skos:narrower ?t . }
}
```
2. Find the root nodes and give them type `pt:Group`, and add `skos:member` predicates toremove the type `skos:Concept`
```
INSERT { ?s a pt:Group , skos:Collection ;  skos:member ?u .}
WHERE {
	?s a skos:Concept ; 
		skos:narrower ?u .
	NOT EXISTS { ?s skos:broader ?t . }
}

DELETE { ?s a skos:Concept ;  skos:narrower ?u .}
WHERE {
	?s a skos:Concept ; 
		skos:narrower ?u .
	NOT EXISTS { ?s skos:broader ?t . }
}
```
3. Find the middle layer and make them `pt:Category`
```
INSERT { ?s a pt:Category , skos:Collection ; skos:member ?u .}
WHERE {
	?s a skos:Concept ; 
	    skos:broader ?t ;
	    skos:narrower ?u . 
}

DELETE { ?s a skos:Concept ; 	 skos:broader ?t ;  skos:narrower ?u .}
WHERE {
	?s a skos:Concept ; 
	    skos:broader ?t ;
	    skos:narrower ?u . 
}
```
Fix up type of `pt:UNCLASSIFIED`
and fix up the `skos:topConceptOf` relationships 
```
DELETE { ?s skos:topConceptOf ?c }
WHERE {
	{ ?s a pt:Group } UNION {?s a pt:Category }
	?s skos:topConceptOf ?c 
}
INSERT { <http://pid.geoscience.gov.au/def/voc/ga/PlaceType> skos:hasTopConcept ?s }
WHERE {
	?s a skos:Concept .
}
```

**Comment by ibastrak** [comment]
returning to this conversation

( content moved to https://github.com/GeoscienceAustralia/Place-Names/issues/25#issuecomment-630593926 )

**Comment by dr-shorthair** [comment]
Current version of the [Place Type classification in RVA](https://vocabs.ands.org.au/viewById/260). The terminology is structured into a taxonomy e.g. 

TERRAIN
&boxur; LANDFORM
&nbsp; &nbsp; &boxur; BANK
&nbsp; &nbsp; &boxur; BAR
&nbsp; &nbsp; &boxur; ...
&nbsp; &nbsp; &boxur; MOUNTAIN
&nbsp; &nbsp; &boxur; ...
&boxur; UNDERSEA
&nbsp; &nbsp; &boxur; ...
&boxur; UNDER_ICE
&nbsp; &nbsp; &boxur; ...

These are **all** encoded as `skos:Concept`. That is as it should be if they all to be used as classifiers directly: e.g. if [Uluru / Ayers Rock](http://ec2-52-63-73-113.ap-southeast-2.compute.amazonaws.com/placenames-dataset/placename/NT_10532) must be classified as ( MOUNTAIN , LANDFORM , TERRAIN )

But it seems to me that 
1. Uluru / Ayers Rock is a MOUNTAIN 
2. MOUNTAIN is a member of the LANDFORM category
3. LANDFORM is a member of the TERRAIN group

i.e. every MOUNTAIN is a LANDFORM, and every LANDFORM is a TERRAIN - you can find that out by looking up [MOUNTAIN](https://vocabs.ands.org.au/repository/api/lda/ga/place-type/v1-0/resource?uri=http://pid.geoscience.gov.au/def/voc/ga/PlaceType/MOUNTAIN) and [LANDFORM](https://vocabs.ands.org.au/repository/api/lda/ga/place-type/v1-0/resource?uri=http://pid.geoscience.gov.au/def/voc/ga/PlaceType/LANDFORM) and [TERRAIN](https://vocabs.ands.org.au/repository/api/lda/ga/place-type/v1-0/resource?uri=http://pid.geoscience.gov.au/def/voc/ga/PlaceType/TERRAIN). So it is redundant to reproduce the whole path-from-the-classification-taxonomy within the record for each individual place like Uluru / Ayers Rock.


BTW - the GA PID service appears to be down at present - so http://pid.geoscience.gov.au/def/voc/ga/PlaceType/LANDFORM is giving a **404**. It should redirect to https://vocabs.ands.org.au/repository/api/lda/ga/place-type/v1-0/resource?uri=http://pid.geoscience.gov.au/def/voc/ga/PlaceType/LANDFORM .

**Comment by dr-shorthair** [comment]
It would be useful to clarify this, then we can consider whether the taxonomy should be encoded as 

`Collection`  -member&rarr; `Concept`

or as 

`Concept` -narrower&rarr; `Concept`
